README.md

```diff
@@ -114,20 +119,21 @@ that directory called `constants.yaml`. You may override these if desired.
     // Whether or not to write your generated schemas to the console.
     preview: false
 
-    // What the root of all of your schemas is. This will make up the first part of
-    // the key of your schemas. The default should be correct.
-    baseUrl: 'http://schema.taskcluster.net/'
+    // The root of the taskcluster cluster
+    rootUrl: 'http://schema.taskcluster.net/'
```

Comment:
bad example?

Reply:
We should probably use something consistent, such as https://taskcluster.example.com`, for example rootUrls.